src/main/java/me/zeroeightsix/kami/module/modules/misc/AutoReconnect.kt

```diff
@@ -16,10 +16,12 @@ import net.minecraft.client.multiplayer.ServerData
         alwaysListening = true
 )
 object AutoReconnect : Module() {
-    private val delay = register(Settings.integerBuilder("Delay").withValue(5000).withRange(100, 10000).withStep(100))
+    private val delay = register(Settings.floatBuilder("Delay").withValue(5.0f).withRange(1.0f, 100.0f).withStep(0.5f))
 
     private var prevServerDate: ServerData? = null
 
+    private var sToMs = 1000.0f
```

Comment:
you're only using this twice, it doesn't hurt to just write out the 1000 twice :p

Reply:
yeah but who knows what could change with this module later on, maybe for some reason this conversion will start to get used more eventually in this module.